test/workerd/tests.mjs

```diff
@@ -60,5 +58,18 @@ export const buffer_implements = {
     }
     assert.ok(new buffer.File([], "file"));
     assert.ok(new buffer.Blob([]));
+    assert.strictEqual(typeof buffer.INSPECT_MAX_BYTES, "number");
+    assert.strictEqual(typeof buffer.resolveObjectURL, "function");
+  },
+};
+
+// --- node:sys
+
+export const sys_implements = {
```

Comment:
Same here.

Actually maybe the only thing we need to test for both modules is that

```
await import("unenv/runtime/node/buffer") === createRequire('/')('node:buffer');
```

Reply:
I don't follow. Can you recommend a `code suggestion` on the github UI?

Reply:
The idea here is that `require("node:buffer)` would be processed by unenv while `createRequire('/')('node:buffer')` would return the actual workerd module.

So it would test that unenv does not change the workerd buffer module.

Reply:
Ah, makes sense. Thank you!